conda_smithy/templates/README.md.tmpl

```diff
@@ -10,7 +10,7 @@ Home: {{ package.meta.about.home }}
 
 Package license: {{ package.meta.about.license }}
 
-Feedstock license: BSD-3-Clause
+Feedstock license: [BSD-3-Clause](LICENSE.txt)
```

Comment:
Doesn't this need to be a full URL?

Reply:
No it's a relative URL based on the feedstock's root directory.

Reply:
That seems too github specific, to be honest. It wouldn't be helpful if someone opened a file locally

Reply:
With markdown a relative local link also workels, see https://stackoverflow.com/questions/32563078/how-link-to-any-local-file-with-markdown-syntax/58317580#58317580

Reply:
How about this? Verified to work with [MacDown]( https://macdown.uranusjr.com ) locally.

```suggestion
Feedstock license: [BSD-3-Clause](./LICENSE.txt)
```

Reply:
Actually it works without the `./` locally. So I don't know that it is needed actually.

Reply:
Ok, so when it works without `./` just let it be as it is

Reply:
I'd rather this be a full URL, personally

Reply:
Sounds good. Will wait to see if Anthony has any other thoughts here :)

Reply:
Updated the PR to use a full link

Reply:
Well that can be tricky if the repo name doesn't match the package name, which does happen when we rename things periodically. Am a little worried about us generating broken links.

Reply:
The link generation uses `github.repo_name` which is used more in the README.md, renaming the packaging would keep the same link, only when you change the repo name the links are broken, but that are at that moment all links to github

Reply:
Yeah, this looks good to me

Reply:
Could you please test this change on this feedstock ( https://github.com/conda-forge/python-feedstock )?

Reply:
Made one fix, attached generated file